content/en/tests/explorer/search_syntax.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Test Visibility Explorer Search Syntax
+kind: documentation
```

Comment:
```suggestion
kind: documentation
aliases:
- /continuous_integration/explorer/search_syntax
```

Same question about this one

Reply:
I don't believe so because we are creating net-new pages. We aren't deleting any pages, which would warrant us adding an alias to the new one.